tests/functional/test_navigation.py

```diff
@@ -9,24 +9,25 @@
 
 @pytest.mark.nondestructive
 def test_navigation(base_url, selenium):
-    page = HomePage(selenium, base_url).open()
-    internet_health_page = page.navigation.open_internet_health()
+    locale = 'de'
```

Comment:
I'm switching these tests to run against `de`, to make sure we don't accidentally break the old navigation whilst it's still around.